From: Etienne Millon <etienne.millon@gmail.com>
Date: Wed, 25 May 2011 17:10:51 +0200
Subject: Fix typo : extention -> extension

This patch fixes a few occurrences of this typo. One was exposed to users via a
warning message.
---
 src/MetaData/metadata.c                  |   12 ++++++------
 src/MetaData/metadata.h                  |    2 +-
 src/browsers/playlist3-playlist-editor.c |    2 +-
 src/plugin.c                             |    2 +-
 src/vapi/gmpc.vapi                       |    2 +-
 5 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/src/MetaData/metadata.c b/src/MetaData/metadata.c
index b665919..5348a3b 100644
--- a/src/MetaData/metadata.c
+++ b/src/MetaData/metadata.c
@@ -982,7 +982,7 @@ gchar * gmpc_get_metadata_filename(MetaDataType  type, mpd_Song *song, char *ext
 	{
 		GError *error = NULL;
 		gchar *filename = NULL, *dirname = NULL;
-		const gchar *extention= (type&(META_ALBUM_TXT|META_ARTIST_TXT|META_SONG_TXT|META_SONG_GUITAR_TAB))?"txt":((ext == NULL)?((type&(META_ALBUM_ART|META_ARTIST_ART))?"jpg":""):ext);
+		const gchar *extension= (type&(META_ALBUM_TXT|META_ARTIST_TXT|META_SONG_TXT|META_SONG_GUITAR_TAB))?"txt":((ext == NULL)?((type&(META_ALBUM_ART|META_ARTIST_ART))?"jpg":""):ext);
 
 		/* Convert it so the filesystem likes it */
 		/* TODO: Add error checking */
@@ -1030,23 +1030,23 @@ gchar * gmpc_get_metadata_filename(MetaDataType  type, mpd_Song *song, char *ext
 			gchar *temp ;
 			g_assert(song->album != NULL);
 			temp =g_filename_from_utf8(song->album,-1,NULL,NULL,NULL); 
-			filename = g_strdup_printf("%s.%s", temp,extention);
+			filename = g_strdup_printf("%s.%s", temp,extension);
 			g_free(temp);
 		}else if(type&META_ARTIST_ART){
-			filename = g_strdup_printf("artist_IMAGE.%s", extention);
+			filename = g_strdup_printf("artist_IMAGE.%s", extension);
 		}else if (type&META_ARTIST_TXT){
-			filename = g_strdup_printf("artist_BIOGRAPHY.%s", extention);
+			filename = g_strdup_printf("artist_BIOGRAPHY.%s", extension);
 		}else if (type&META_SONG_TXT) {
 			gchar *temp ;
 			g_assert(song->title != NULL);
 			temp =g_filename_from_utf8(song->title,-1,NULL,NULL,NULL); 
-			filename = g_strdup_printf("%s_LYRIC.%s", temp,extention);
+			filename = g_strdup_printf("%s_LYRIC.%s", temp,extension);
 			g_free(temp);
 		}else if (type&META_SONG_GUITAR_TAB) {
 			gchar *temp ;
 			g_assert(song->title != NULL);
 			temp =g_filename_from_utf8(song->title,-1,NULL,NULL,NULL); 
-			filename = g_strdup_printf("%s_GUITAR_TAB.%s", temp,extention);
+			filename = g_strdup_printf("%s_GUITAR_TAB.%s", temp,extension);
 			g_free(temp);
 		}
 		filename = strip_invalid_chars(filename);
diff --git a/src/MetaData/metadata.h b/src/MetaData/metadata.h
index 4b0cb0f..625263a 100644
--- a/src/MetaData/metadata.h
+++ b/src/MetaData/metadata.h
@@ -142,7 +142,7 @@ void meta_data_destroy(void);
 MetaDataResult meta_data_get_path(mpd_Song *tsong, MetaDataType type, MetaData **met,MetaDataCallback callback, gpointer data);
 
 
-gchar * gmpc_get_metadata_filename(MetaDataType  type, mpd_Song *song, char *extention);
+gchar * gmpc_get_metadata_filename(MetaDataType  type, mpd_Song *song, char *extension);
 
 
 
diff --git a/src/browsers/playlist3-playlist-editor.c b/src/browsers/playlist3-playlist-editor.c
index 512e58c..53db23f 100644
--- a/src/browsers/playlist3-playlist-editor.c
+++ b/src/browsers/playlist3-playlist-editor.c
@@ -66,7 +66,7 @@ static void playlist_editor_set_enabled(int enabled)
 
 
 /**
- * Browser extention 
+ * Browser extension
  */
 gmpcPlBrowserPlugin playlist_editor_gbp = {
   .add = playlist_editor_browser_add,   /* Add */
diff --git a/src/plugin.c b/src/plugin.c
index a002f2b..2d66ef2 100644
--- a/src/plugin.c
+++ b/src/plugin.c
@@ -320,7 +320,7 @@ void plugin_load_dir(const gchar *path)
             else
             {
                 g_log(PLUGIN_LOG_DOMAIN, G_LOG_LEVEL_INFO,
-                        "%s not loaded, wrong extention, should be: '%s'", dirname,G_MODULE_SUFFIX);
+                        "%s not loaded, wrong extension, should be: '%s'", dirname,G_MODULE_SUFFIX);
             }
             q_free(full_path);
         }
diff --git a/src/vapi/gmpc.vapi b/src/vapi/gmpc.vapi
index 850f1d3..7fa39a3 100644
--- a/src/vapi/gmpc.vapi
+++ b/src/vapi/gmpc.vapi
@@ -121,7 +121,7 @@ namespace Gmpc {
         public void set_metadata(MPD.Song song, Result result, Gmpc.MetaData.Item met); 
 
         [CCode ( cname="gmpc_get_metadata_filename", cheader_filename="libmpd/libmpd.h,metadata.h")]
-        public string get_metadata_filename(Type type, MPD.Song song, string? extention);
+        public string get_metadata_filename(Type type, MPD.Song song, string? extension);
 
         [CCode ( cname="GmpcMetaImage", cheader_filename="gmpc-metaimage.h")]
         public class Image: Gtk.Widget {
-- 
